/**:
  ros__parameters:
    mask_topic: /event_mask
    pointcloud_topic: /dynamic_obstacles
    marker_topic: /dynamic_obstacles_markers
    eps_pixels: 6.0
    min_samples: 5
    max_points: 15000
    pixel_scale_m: 0.01
    use_ground_plane: true
    camera_frame: event_camera
    output_frame: base_link
    fx: 264.69243200992327
    fy: 264.54451579953479
    cx: 183.38023825181807
    cy: 126.8117161671034
    k1: -0.4134720187444611
    k2: 0.27323002022969767
    p1: -0.0008530084452305791
    p2: 0.00100858047727728
    k3: -0.129631949994604
    undistort: true
    undistort_iterations: 5
    plane_z: 0.0
    use_latest_tf: true
    publish_centroids_only: true
    log_stats: true
    use_depth: true
    depth_topic: /aps_depth
    depth_scale: 1.0
    depth_min: 0.1
    depth_max: 10.0
    depth_use_median: true
